readthedocs/settings/base.py

```diff
@@ -495,7 +495,7 @@ def _get_docker_memory_limit(self):
                 "free -m | awk '/^Mem:/{print $2}'",
                 shell=True,
             ))
-            return total_memory, round(total_memory - 750, -2)
+            return total_memory, round(total_memory - 1000, -2)
```

Comment:
This should probably be a constant, or at least have a comment explaining the logic.

Reply:
Agree. I'm merging it for now so it goes out tomorrow, tho.